PukingPredator/Assets/Scripts/Player.cs

```diff
@@ -132,4 +140,25 @@ private IEnumerator SetGroundedFalse(float delay)
         ungroundedCoroutine = null;
     }
 
+    private void AlterMovement(Item item, int isEating)
+    {
+        switch (item.size)
+        {
+            case ItemSize.small:
+                moveSpeed -= 1f * isEating;
+                jumpForce -= 1f * isEating;
```

Comment:
if were just doing multiplication based on the value of the enum, converting this to use a dictionary or pulling out the value to somewhere else more intuitive might be better